share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -78,22 +78,38 @@ default:
       when: never
     - if: $SPACK_CI_ENABLE_STACKS =~ /.+/ && $SPACK_CI_STACK_NAME !~ $SPACK_CI_ENABLE_STACKS
       when: never
-    - if: $CI_COMMIT_REF_NAME == "develop" || $CI_COMMIT_REF_NAME =~ /^releases\/v.*/
+    - if: $CI_COMMIT_REF_NAME == "develop"
```

Comment:
Yeah, keeping `develop` pipelines going makes sense to me, in order to keep the PR pipelines running smoothly (or at least no less smoothly :laughing: than currently).